src/views/app/detail-panel/conversation-preview-panel.jsx

```diff
@@ -50,7 +50,7 @@ const MessagesComponent = ({ conversation }) => {
 		return map(convMessages, (message, index) =>
 			message ? (
 				<Padding
-					key={`${conversationId}-${message.id}-${message.date}`}
+					key={`${conversationId}-${message.id}-${message.date}-${index}`}
```

Comment:
this key is becoming a bit excessive, could it be simplified with only id and index? (using the index in the key opens up to a lot of issues, but i trust you already know it)

Reply:
@zovomat noted and reduced to conversationId-messageId